mtags/src/main/scala-3/scala/meta/internal/pc/completions/CompletionValue.scala

```diff
@@ -154,7 +154,7 @@ object CompletionValue:
     override def completionItemKind(using Context): CompletionItemKind =
       CompletionItemKind.File
 
-  case class ScalaCLiImport(
+  case class IvyImport(
```

Comment:
Can we have one `CompletionValue` for both ScalaCli and Ammonite dependency imports?

Reply:
Sure, it's probably better, we could even hoave just one for sbt as well.

Reply:
ach right :sweat_smile: